packages/katello/katello/hostname-change.rb

```diff
@@ -143,8 +143,7 @@ def next_steps_message
   You will have to install the new bootstrap rpm and reregister all clients and #{@plural_proxy} with subscription-manager
   (update organization and environment arguments appropriately):
 
-  yum remove -y katello-ca-consumer*
-  rpm -Uvh http://#{@new_hostname}/pub/katello-ca-consumer-latest.noarch.rpm
+  set -o pipefail && curl -sS "http://#{@new_hostname}/unattended/public/foreman-ca-refresh" | bash
   subscription-manager register --org="Default_Organization" --environment="Library" --force
 
   Then reattach subscriptions to the client(s) and run:
```

Comment:
attach is no longer necessary with SCA?

Reply:
@ianballou any thoughts on this one? I don't mind doing the change, but open to suggestions about what is needed here.